cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -5827,11 +5827,85 @@ search = (`cs_mssql` (sourcetype="mssql:audit" OR sourcetype="mssql:audit:json")
+action.cyences_send_email_action = 1
+action.cyences_notable_event_action.products = MSSQL
+
+[MSSQL - Role Changes]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 33 * * * *
+description = An alert for role changes \
+\
+Data Collection - Splunk DB Connect and Splunk Add-on for Microsoft SQL Server (https://splunkbase.splunk.com/app/2648).
+dispatch.earliest_time = -62m@m
+dispatch.latest_time = -2m@m
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = (`cs_mssql` (sourcetype="mssql:audit" OR sourcetype="mssql:audit:json")) OR (`cs_windows_idx` source="WinEventLog:Application" EventCode=33205 ) class_type_desc IN("SERVER ROLE", "ROLE")  _index_earliest=-60m@m _index_latest=now  | rename properties.* as *  \
+| eval status = if(succeeded==1 or succeeded=="true", "success", "failure") \
+| rename server_instance_name as db_host, server_principal_name as user, class_type_desc as object_type, client_ip as src, database_principal_name as db_user, application_name as client_app \
+| `cs_human_readable_time_format(_time, event_time)` \
+| table event_time cyences_severity host db_host user action_name schema_name database_name object_name object_type src db_user client_app statement status \
+| eval cyences_severity="high" \
+| `cs_mssql_role_changes_filter`
+action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.param.filter_macro_name = cs_mssql_role_changes_filter
+action.cyences_notable_event_action.contributing_events = (`cs_mssql` (sourcetype="mssql:audit" OR sourcetype="mssql:audit:json")) OR (`cs_windows_idx` source="WinEventLog:Application" EventCode=33205 ) class_type_desc IN("SERVER ROLE", "ROLE") | rename properties.* as *
+action.cyences_notable_event_action.system_compromised_search = | stats count by host
+action.cyences_notable_event_action.system_compromised_drilldown = (`cs_mssql` (sourcetype="mssql:audit" OR sourcetype="mssql:audit:json")) OR (`cs_windows_idx` source="WinEventLog:Application" EventCode=33205 ) class_type_desc IN("SERVER ROLE", "ROLE") host=$row.host$ | rename properties.* as *
+action.cyences_notable_event_action.attacker_search = | stats count by user
+action.cyences_notable_event_action.attacker_drilldown = (`cs_mssql` (sourcetype="mssql:audit" OR sourcetype="mssql:audit:json")) OR (`cs_windows_idx` source="WinEventLog:Application" EventCode=33205 ) class_type_desc IN("SERVER ROLE", "ROLE") user=$row.user$ | rename properties.* as *
+action.cyences_send_email_action = 1
+action.cyences_notable_event_action.products = MSSQL
```

Comment:
Just confirm the alert category with SOC and Compliance team and let me know. 
I'll add categories (SOC or Compliance or both) to all remaining alerts before the app release.

Reply:
I think its Compliance, but Banking Institutes put it as part of Security in general. But we can discuss in the meeting and put it in that way.